<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanelFormWidget</class>
 <widget class="QWidget" name="AdminPanelFormWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>761</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Панель администратора</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="showUsersTab">
       <attribute name="title">
        <string>Просмотр пользователей</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="addUsersTab">
       <attribute name="title">
        <string>Добавление пользователей</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>20</y>
          <width>721</width>
          <height>518</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="nameFormLayout">
           <item row="1" column="1">
            <widget class="QLineEdit" name="nameLineEdit">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="nameLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Имя</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="statusLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>28</pointsize>
              </font>
             </property>
             <property name="text">
              <string>STATUS LABEL</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="lastnameFormLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="lastNameLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Фамилия</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lastNameLineEdit">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="middleNameFormLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="middleNameLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Отчество</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="middleNameLineEdit">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="loginFormLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="loginLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Логин</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="loginLineEdit">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="passwordFormLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="passwordLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Пароль</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="passwordLineEdit">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="phoneNumberFormLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="phoneNumberLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Номер телефона</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="phoneNumberLineEdit">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="emailFormLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="emailLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="emailLineEdit">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="passFormLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="passLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Паспортные данные</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="passLineEdit">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="roleFormLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="roleLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Роль</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="roleComboBox">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>Администратор</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Инструктор</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ученик</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="addUserPushButton">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>560</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить пользователя</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="manageScheduleTab">
       <attribute name="title">
        <string>Управление расписанием</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="manageTestsTab">
       <attribute name="title">
        <string>Управление тестами</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
